### PR TITLE
Add note about restarting OMERO.web

### PR DESCRIPTION
**Pull request by hflynn**
In response to QA 17257 where a user commented that it doesn't say you need to restart OMERO.web on the customization page.

Staged at https://www.openmicroscopy.org/site/support/omero5.3-staging/sysadmins/unix/install-web.html#customizing-your-omero-web-installation and https://www.openmicroscopy.org/site/support/omero5.3-staging/sysadmins/customization.html

### CODE REVIEW

File: omero/sysadmins/unix/install-web.txt [hunk 1]
```diff
@@ -176,3 +176,7 @@ The most popular configuration options include:
 -  Customizing index page, see :property:`omero.web.index_template`.
 
 -  Enabling a public user see :doc:`/sysadmins/public`.
+
+.. Note:: Depending on how your OMERO.web server is deployed, you may need to
```
Comment (atarkowska) [inline omero/sysadmins/unix/install-web.txt]: OMERO.web is an app not a server

Comment (atarkowska) [inline omero/sysadmins/unix/install-web.txt]: I would be more explicit. something like: Depending on the deployment choice OMERO.web will not activate the configuration changes until either restarting gunicorn using `bin/omero web restart` (nginx) or reloading Apache webserver.

Comment (hflynn) [inline omero/sysadmins/unix/install-web.txt]: Thanks Ola, I'll push a fix shortly

**Comment by hflynn** [comment]
Rebuilding staging now

**Comment by joshmoore** [comment]
As a start, :+1:. I do wonder if we're getting lost in "Note:" boxes though especially on customization.html. (3 boxes with more vertical space than the introductory text). It might be that we need an intro to customization that shows the general steps, but that may be part of a larger restructuring. (At which point the install-web.html "Customizing" section could be folded under?)

**Comment by hflynn** [comment]
General tidy up on staging now, have cut down the note boxes on customization page and rejigged the section at the bottom of the web deployment page too.

**Comment by joshmoore** [comment]
:+1: from me. Still think there's room for us to handle more rejigging, but I think that's beyond the scope of this PR.

**Comment by hflynn** [comment]
Agreed.

Once we are clear on our recommendations for deploying OMERO.web for 5.3, these sections could probably do with an overhaul similar to the installation docs - I'll add it to a card.

**Comment by hflynn** [comment]
--rebased-to #1505

**Pull request merged**